main.js

```diff
@@ -0,0 +1,87 @@
+const Board = function (x) {
+x = [["1", "2", "3"],
+["4", "5", "6"],
+["7", "8", "9"]];
+return x;
+}
+
+var gameBoard;
+var playerTurn;
+var moveCount = 0;
+
+function printBoard() {
+  var output = gameBoard[0].join("") + "\n" + gameBoard[1].join("") + "\n" + gameBoard[2].join("");
+  console.log(output);
+}
+
+function newGame() {
+  moveCount = 0;
+  playerTurn = "O"
+  gameBoard = new Board;
```

Comment:
No need to use `new` here as `Board` returns and object anyway and isn't utilizing `this` inside of it.